tests/unit/test_s3.py

```diff
@@ -0,0 +1,204 @@
+from unittest.mock import Mock, patch
+
+import pandas as pd
+
+from sdgym.s3 import is_s3_path, parse_s3_path, write_csv, write_file
+
+
+def test_is_s3_path_with_local_dir():
+    """Test the ``sdgym.s3.is_s3_path`` function with a local
+    directory. Expect that the return value is false.
+
+    Input:
+    - path
```

Comment:
I would try to be a bit more verbose about the input and output specification. More than just the type, we should say what is being tested (what is being passed) and how it is being validated (what do you expect to happen).

Here I would say:
```
Input:
- path to a local file
Output:
- False
```